### PR TITLE
updated beaker imports to match the other sourcecodes for consistency

### PR DESCRIPTION
**Pull request by iskysun96**
Since most code imports specific methods and libraries from beaker, I changed the import for consistency.

### CODE REVIEW

**Comment by barnjamin** [comment]
thanks! assume you ran pytest to make sure it all works the same?

**Comment by iskysun96** [comment]
I did but I got the following error for both before and after I made the code changes.

image

**Comment by barnjamin** [comment]
You'll need to update your pyteal version to >= 0.17.0, also merge master back into your branch before trying again

**Comment by iskysun96** [comment]
Is pytest implemented for this file? Don't see any tests defined.

**Comment by barnjamin** [comment]
just the regular testing from root with pytest should be sufficient

**Comment by iskysun96** [comment]
Are all tests successful on your side right now?

File: beaker/testing/unit_testing_helpers.py [hunk 1]
```diff
@@ -81,14 +81,16 @@ def assert_output(
...
+        algod_client = sandbox.get_algod_client()
 
-    if accounts is None:
-        accounts = bkr.sandbox.get_accounts()
+    if sandbox_accounts is None:
+        sandbox_accounts = sandbox.get_accounts()
 
-    app_client = bkr.client.ApplicationClient(client, app, signer=accounts[0].signer)
+    app_client = client.ApplicationClient(
+        client, app, signer=sandbox_accounts[0].signer
```
Comment (barnjamin) [inline beaker/testing/unit_testing_helpers.py:92]: still have client here

Comment (iskysun96) [reply beaker/testing/unit_testing_helpers.py:92]: good catch.

**Comment by iskysun96** [comment]
getting this error for some reason. I'm just running assert_output here. Also getting this error for other test files in beaker
image

**Pull request merged**